### DISCUSSION TITLE
/installing-valetudo-on-xiaomi-roborock-s5-robovac-the-easy-way/

### DISCUSSION THREAD

**Discussion by migratecomments** [ironicbadger/blog.ktz.me#148] [Announcements]
Installing Valetudo on Xiaomi Roborock S5 Robovac - The easy way!
        
[https://blog.ktz.me/installing-valetudo-on-xiaomi-roborock-s5-robovac-the-easy-way/](https://blog.ktz.me/installing-valetudo-on-xiaomi-roborock-s5-robovac-the-easy-way/)

**Comment by 0xtavi** [comment]
Hi there, the tutorial was great. I've easily went through all the steps and got valetudo on my roborock. Thank you for this!

I want to SSH into the roborock but I can't find out how. Is there any default password set after the flash is finished?
